### DISCUSSION TITLE
Would this be an ImportExcel function call?

### DISCUSSION THREAD

**Discussion by dwchan69** [dfinke/ImportExcel#1570] [General]
Let say I have 2 sheets in a workbook
- sheet 1 contain information to an item based on size (small, medium, large)  So for the entries for a rows, it may say small, $1.00, 6' long, For the next row, it may say medium $2.00, 12' long
- sheet 2 contain the order.  Row 1 may say, 2 units, small, , Row 2 may say 4 unit, medium .etc> 

I am able to pull in the data individually on each sheet using something like this
$Orders = Import-Excel -Path $DataSourcePath -WorksheetName 'Order', and get each column cells info by referring $Order."First row column cell name"

What I am curious is how I link the 2 sheets together so when I pull the 1st Order, it not only pull number of unit from Order 1, it knows it is a small, but it also pull the small product information with it, something like

2 unit, small, $1.00, 6' long.

**Comment by dfinke** [comment]
You'll need to write all of that in PowerShell. Using `Import-Excel` to read both sheets into separate variables and then writing lookup code and outputting the results.
